doc/result_listener.md

```diff
@@ -0,0 +1,15 @@
+# Result listener
+Add the result listener called `Parquet` to your test plan.
```

Comment:
Did you mean settings here?

Also I think we should start by giving a general overview of the Parquet result listener.  
Then write how to use it and finally get into the details.

Reply:
Let me know if the most recent changes resolve this comment